README.md

```diff
@@ -179,7 +179,8 @@ optional arguments:
                         Description of the source of the evidence
   -n NAME, --name NAME  Descriptive name of the evidence
 ```
-
+Sample rawdisk command:
+`turbiniactl rawdisk -l {PATH_TO_THE_IMAGE}`
```

Comment:
Maybe put another newline here so it separates these?  The rendered page puts this on the same line as the next sentences, and that may not be what you intended.